turborepo-integration-tests/turbo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": ["//"],
+  "pipeline": {
+    "test": {
+      "dependsOn": ["cli#build"],
```

Comment:
This is here for completeness, but in the general case, we're building a local turbo before we run the `test` task, so turbo is already built. I can remove this if people have strong opinions.